modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePaymentTerms.testcase

```diff
@@ -0,0 +1,57 @@
+@component-name = "portal-commerce"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Order Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		Navigator.openURL();
+
+		if (IsElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE")) {
+			User.logoutPG();
+		}
+
+		User.firstLoginPG();
+
+		CommerceConfiguration.commerceAdminTearDown();
+
+		CommerceConfiguration.commerceInitializersTearDown();
+	}
+
+	@description = "Delete a new payment term to Terms and Conditions."
+	@priority = "3"
+	test DeleteAPaymentTerm {
+		property portal.acceptance = "false";
+
```

Comment:
Please add Task blocks. This helps analysts review the tests. See [example here](https://github.com/liferay/liferay-portal/blob/master/modules%2Fapps%2Fcommerce%2Fcommerce-product-test%2Fsrc%2FtestFunctional%2Ftests%2FCPCommerceOrderTypes.testcase#L294-L312) and the resulting [Poshi summary here](https://testray.liferay.com/reports/production/logs/2022-03/test-1-7/test-portal-testsuite-upstream(master)/840/functional-tomcat90-mysql57-jdk8/21/3/LocalFile.CPCommerceOrderTypes_DeleteOrderType/summary.html.gz?authuser=0#).
For this test, the blocks will be similar to what's on the ticket:
```
task ("Given a new entry into the Terms and Conditions with the status to Active) {
    lines 33-48
}

task ("When I delete the terms and conditions") {
    line 50
}

task ("Then the entry is delete) {
    line 52-54
}
```